go/pkg/tool/embeddedtool.go

```diff
@@ -119,7 +121,7 @@ var RemoteToolOperations = map[OpType]func(ctx context.Context, c *Client){
 		}
 	},
 	uploadDir: func(ctx context.Context, c *Client) {
-		us, err := c.UploadDirectory(ctx, getPathFlag())
+		us, err := c.UploadDirectory(ctx, getPathFlag(), getSymlinkBehaviorType())
```

Comment:
nit: use the function you added to Command to parse the string as enum.

Reply:
Is it okay to add the dependency on `command` here? I didn't want to introduce a new dep unnecessarily and potentially break some boundaries. Will do!